custom_components/xiaomi_miio_airpurifier/fan.py

```diff
@@ -380,6 +409,12 @@
     "High",
     "Strong",
 ]
+OPERATION_MODES_AIRPURIFIER_MIOT = [
+    "Auto",
+    "Silent",
+    "Favorite",
+    "Fan" # TODO: Manual?
```

Comment:
Likewise with this todo. I think it had something to do with the wording in the app.